src/handles/handleFormat.ts

```diff
@@ -38,7 +47,11 @@ async function handleFormat(
     }
   )
   let output = '';
-  if (isStdout === undefined && isStderr === undefined) {
+  if (code > 0) {
```

Comment:
The stderr / stdout should probably be logged to a file. Also another property like `IFormatterConfig.skipOnError` couldn't hurt.